### ISSUE TITLE
Numbers are not visible on some spin button items.(Windows11 Widget Theme and HiDPI?)

### ISSUE THREAD

**Issue by nanitaro** [rncbc/qsynth#81]
Hello.

Windows 11 (Surface 7 Pro , Custom scaling set to 200%)
Qsynth 0.9.90

In the above environment, numbers are not visible on some spin button items.
For example, MIDI Channels in the Setup screen, MIDI tab.
![setup-midi-0 9 90](https://github.com/rncbc/qsynth/assets/50589546/4ce72171-84fd-4f46-af10-7f1a464a6100)
Polyphony in the Audio tab.
![setup-audio 0 9 90](https://github.com/rncbc/qsynth/assets/50589546/c1a8b3a8-a3f6-4f30-811c-3d7e6bf9ba93)

In 0.9.13, they were displayed without any problem.
![setup-midi-0 9 13](https://github.com/rncbc/qsynth/assets/50589546/f46fa78b-3f5e-4ece-8f64-06ef7c25d067)
![setup-audio 0 9 13](https://github.com/rncbc/qsynth/assets/50589546/60d5d8eb-6360-4d7b-a7f2-1063ed9bd402)

**Comment by pedrolcl** [comment]
I cannot reproduce this issue in Windows 10, with my screen (125% scaling).

In my opinion, this should be fixed in Qsynth, with a more flexible layout setting for spin boxes.

But anyway, here is a workaround for you: 
* Press the "Options" button in the main window.
* Switch to the "Display" tab
* in the combo box "Widget style theme", try some other option. My suggestion is "Fusion"

![imagen](https://github.com/rncbc/qsynth/assets/15184708/1aad0a8c-e813-4436-a605-31e879b9018b)

**Comment by nanitaro** [comment]
Thank you.
I set the Widget style theme to something other than Default and Windows11 and that solved the problem.
![setup-midi-0 9 90-fusion](https://github.com/rncbc/qsynth/assets/50589546/6c020cae-975a-4bc9-b557-6ae6551f81d1)

I understand that the problem is with the Windows11 theme.
I will leave it open until the problem is resolved.

**Comment by pedrolcl** [comment]
> I understand that the problem is with the Windows11 theme.

Yes. The [Windows11 style](https://bugreports.qt.io/browse/QTBUG-97668) is new in Qt 6.7 (recently released), and it has been bundled in Qsynth 0.9.90.  The spin boxes are more "touch friendly" than before, but they also demand more screen space. This is only a windows issue for now, but having style plugins in Qt means that we may have this problem as well in the future in a new KDE or GTK style.

@rncbc :  what do you think about changing the spin boxes in `qsynthSetupForm.ui` , sizePolicy - horizontal, from Minimum to MinimumExpanding? 

Example:

```
     <widget class="QSpinBox" name="MidiChannelsSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding"
        [...]
```

**Comment by nanitaro** [comment]
Sorry for the delay in responding.
The spin box refurbishment seems to be mostly OK.
(Confirmed with 1.0.0).

However, I had one point of concern.
It is the MIDI device select box.
![setup](https://github.com/user-attachments/assets/bd08d4e0-ff0d-458d-b9f8-aa4cde2a3436)

If there is a long MIDI device name, the number and the beginning of the name seem to be omitted.
I think it is inevitable to omit MIDI device names, but I think it would be easier to understand if the end of the name is omitted rather than the beginning.

If you do not see a need to fix this, close this issue.
